src/lib/evas/canvas/meson.build

```diff
@@ -1,7 +1,49 @@
+pub_eo_files_optional_ector_ = [
+  ['efl_canvas_group.eo', 'efl_canvas_group_vg.eo_template', 'efl_canvas_group_non_vg.eo_template'],
+  ['efl_canvas_image_internal.eo', 'efl_canvas_image_internal_vg.eo_template', 'efl_canvas_image_internal_non_vg.eo_template'],
+  ['efl_canvas_textblock.eo', 'efl_canvas_textblock_vg.eo_template', 'efl_canvas_textblock_non_vg.eo_template'],
+]
+
+foreach eo_file : pub_eo_files_optional_ector_
+  if get_option('ector')
+    in_eo = eo_file[1]
+  else
+    in_eo = eo_file[2]
+  endif
+  eo_file = eo_file[0]
+
+  configure_file(
+    input : in_eo,
+    output : eo_file,
+    copy: true,
+  )
+
+  priv_eo_file_target += custom_target('eolian_gen_' + eo_file,
```

Comment:
Where is this variable initialized?

Reply:
`src/lib/eo/meson.build:28:priv_eo_file_target = []`

and elementary rewrite it:
`src/lib/elementary/meson.build:250:priv_eo_file_target = []`

Reply:
Hm, do we want to append? How does it work/how is it processed?

Reply:
Here I'm just mimicking [this](https://github.com/expertisesolutions/efl/pull/482/files?file-filters%5B%5D=.build&file-filters%5B%5D=.c&file-filters%5B%5D=.cc&file-filters%5B%5D=.eo_non_vg#diff-724e8a6f046442f85b7751564ca86b86dd22fb35b9d5e56b9c3b345b6c14fbb8L62), as it was done before this patch.